src/microbe_stage/MicrobeCamera.cs

```diff
@@ -292,7 +255,7 @@ public void ResetHeight()
     public void SetCustomCurrentStatus(bool current)
     {
         Current = current;
-        UpdateBackgroundVisibility();
+        backgroundPlane.SetVisibility(Current);
```

Comment:
```suggestion
        UpdateBackgroundVisibility();
```

Why doesn't this just call that method that still exists and does the SetVisibility call?